e2e_playwright/shared/app_utils.py

```diff
@@ -222,3 +222,40 @@ def click_checkbox(
     checkbox_element = get_checkbox(page, label)
     checkbox_element.click()
     wait_for_app_run(page)
+
+
+def expect_help_tooltip(
+    app: Page, el_with_help_tooltip: Locator, tooltip_text: str | Pattern[str]
+):
+    """Expect a tooltip to be displayed when hovering over the help symbol of an element.
+
+    This only works for elements that have our shared help tooltip implemented.
+    It doesn't work for elements with a custom tooltip implementation, e.g. st.button.
```

Comment:
I would add here that in the end of the function the element is unhovered.

Reply:
Added a comment 👍